indico/modules/events/editing/models/revisions.py

```diff
@@ -113,8 +113,6 @@ class EditingRevision(RenderModeMixin, db.Model):
         lazy=True,
         backref=db.backref(
             'revisions',
-            primaryjoin=('(EditingRevision.editable_id == Editable.id) & '
-                         f'(EditingRevision.type != {RevisionType.reset})'),
```

Comment:
Removing this filter doesn't cause issues anywhere?

Reply:
well, I hope I squashed most of the problems...